src/methods/create.js

```diff
@@ -559,6 +559,67 @@ export class CreateImplementation {
     return O.$DefineOwnProperty(P, newDesc);
   }
 
+  CopyDataProperties(realm: Realm, target: ObjectValue, source: Value, excluded: Array<StringValue>) {
+    // Assert: Type(target) is Object.
+    invariant(target instanceof ObjectValue, "Not an object value");
+
+    // Assert: Type(excluded) is List.
+    invariant(excluded instanceof Array, "Not an array");
+
+    let keys, from;
+    //   If source is undefined or null, let keys be a new empty List.
+    /* eslint-disable brace-style */
+    if (source === realm.intrinsics.null || source === realm.intrinsics.undefined) {
+      keys = [];
+
+      // Seems necessary. Flow complained too.
+      from = new Value(realm);
+    }
+    /* eslint-enable brace-style */
+    //   Else,
+    else {
+      // Let from be ! ToObject(source).
+      from = To.ToObject(realm, source);
+
+      // Let keys be ? from.[[OwnPropertyKeys]]().
+      keys = from.$OwnPropertyKeys();
+    }
```

Comment:
Move this curly to the end after the loop. That way you don't have to assign anything to `from` since Flow will know that keys is empty. You can also avoid assigning anything to keys and just leave a comment there.